src/engine/sync/syncable.h

```diff
@@ -36,6 +36,8 @@ inline bool isMaster(SyncMode mode) {
     return (mode == SYNC_MASTER_SOFT || mode == SYNC_MASTER_EXPLICIT);
 }
 
+/// Syncable is an abstract base class for any object that wants to participate
+/// in Master Sync.
```

Comment:
I'm a bit confused about the difference between Syncable and BaseSyncableListener. Are there classes that only listen to sync but aren't able to be synced to?

Reply:
hm it looks like currently only EngineSync inherits from basesyncablelistener.  It could probably be combined with EngineSync

Reply:
(updated comment)